app/assets/src/components/Samples.jsx

```diff
@@ -917,7 +917,42 @@ class Samples extends React.Component {
     location.href = `/projects/${id}/csv`;
   }
 
+  applyExcluded(e, type, state_var) {
+    let id = e.target.getAttribute('data-exclude')
+    if (type === "int") {
+      id = +id
+    }
+    let list = this.state[state_var].splice(0);
```

Comment:
use Object.assign instead if you intend to copy the content?

Reply:
ok